### PR TITLE
add feature to change line size with mouse wheel

### PR DESCRIPTION
**Pull request by Ahmadrezadl**

### CODE REVIEW

File: frontend/templates/lobby.html [hunk 1]
```diff
@@ -1610,6 +1610,31 @@
...
+            else if(sizeSlot < 28)
+                n = 24
+            else
+                n = 32
+
+            document.getElementById('size-' + n + '-button').checked = true;
+            setLineWidth(sizeSlot);
+        }
+
+        drawingBoard.onwheel = wheelSize;
```
Comment (Bios-Marcel) [inline frontend/templates/lobby.html:1637]: I'd prefer addEventListener for consistency.

File: frontend/templates/lobby.html [hunk 2]
```diff
@@ -1610,6 +1610,31 @@
 
             return circleData;
         }
+        
+        let sizeSlot = 8;
```
Comment (Bios-Marcel) [inline frontend/templates/lobby.html:1614]: There should already be a variable with the current size. It's called lineWidthUnscaled If I remember correctly.

File: frontend/templates/lobby.html [hunk 3]
```diff
@@ -1610,6 +1610,31 @@
...
+        function wheelSize(e) {
+            event.preventDefault();
+            sizeSlot += Math.min(Math.max(-1, e.deltaY), 1);
```
Comment (Bios-Marcel) [inline frontend/templates/lobby.html:1617]: Why is this necessary if the sizes are fixed numbers?

File: frontend/templates/lobby.html [hunk 4]
```diff
@@ -1610,6 +1610,31 @@
...
+            if (sizeSlot < 6) {
+                sizeSlot = 6;
+            } else if (sizeSlot > 32) {
+                sizeSlot = 32;
+            }
+
+            if(sizeSlot < 12)
+                n = 8
```
Comment (Bios-Marcel) [inline frontend/templates/lobby.html:1625]: Is n even necessary? Could just use `sizeSlot` or furthermore `lineWidthUnscaled`

File: frontend/templates/lobby.html (hunk 3 above)
Comment (Ahmadrezadl) [reply frontend/templates/lobby.html:1617]: sizes are not fixed numbers. 🤔
see https://bekeshbekesh.herokuapp.com/

File: frontend/templates/lobby.html (hunk 1 above)
Comment (Ahmadrezadl) [reply frontend/templates/lobby.html:1637]: i don't know about this :[ sorry

File: frontend/templates/lobby.html (hunk 3 above)
Comment (Bios-Marcel) [reply frontend/templates/lobby.html:1617]: i don't really understand ^^

There's 4 base sizes that have fixed numbers: 8,16,24 and 32

What do you mean?

File: frontend/templates/lobby.html (hunk 1 above)
Comment (Bios-Marcel) [reply frontend/templates/lobby.html:1637]: It's fine, that's why I am telling you ^^

File: frontend/templates/lobby.html (hunk 3 above)
Comment (Ahmadrezadl) [reply frontend/templates/lobby.html:1617]: image
in this feature you can choose something between 8 and 16 ... like 10
but it marks closest brush size on the bottom =) 
you can see the feature on my fork version.

Comment (Bios-Marcel) [reply frontend/templates/lobby.html:1617]: I see, I missunderstood at first. However, how would people with touchscreen achieve the same sizes then?
I'd like to keep features across devices mostly the same. Do you have any idea?

Comment (Ahmadrezadl) [reply frontend/templates/lobby.html:1617]: it's not that important I think
but a text box with "brush size" title? =) 
it's not a good thing I guess
so, nevermind. close the pool request

Comment (Bios-Marcel) [reply frontend/templates/lobby.html:1617]: Generally I don't think it's a bad idea, as I also just added shortcuts for switching. I also want to add a better colorpicker on the canvas itself.

Now you wouldn't be able to set these inbetween sizes on mobile. It's also a bit weird that we'd display one of the radio buttons as checked, even though it's not technically true.

Sorry, just a bit perfectionist here :D

Comment (Ahmadrezadl) [reply frontend/templates/lobby.html:1617]: it's ok =) so good luck
